module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -460,6 +464,58 @@ protected function getItemStatusError($record, $msg = '')
         ];
     }
 
+    /**
+     * Convert availability boolean or int to a string
+     *
+     * @param mixed $available Boolean value or one of the ItemStatus::STATUS_*
+     *                         constants
+     *
+     * @return string
+     */
+    protected function availabilityToString($available): string
+    {
+        switch ((int)$available) {
+            case ItemStatus::STATUS_UNAVAILABLE:
+                return 'false';
+            case ItemStatus::STATUS_UNCERTAIN:
+                return 'uncertain';
+            default:
+                return 'true';
+        }
+    }
+
+    /**
+     * Get a message for availability status
+     *
+     * @param array  $messages         Available messages
+     * @param string $available        String 'false', 'true' or 'uncertain'
+     * @param bool   $useUnknownStatus Whether to use the "unknown status" message
+     *
+     * @return string
+     */
+    protected function getAvailabilityMessage(
+        array $messages,
+        $available,
+        bool $useUnknownStatus
+    ): string {
+        if ($useUnknownStatus) {
+            $key = 'unknown';
+        } else {
+            switch ($available) {
```

Comment:
Is it really worth using a switch here? Would an if..elseif..else be more readable? (I'm okay with keeping it, especially if you expect this to grow significantly in future, but a switch feels like overkill for three options).

Reply:
I agree it's cleaner with an `if` chain, changed.